examples/jsm/loaders/3MFLoader.js

```diff
@@ -1468,7 +1468,7 @@ class ThreeMFLoader extends Loader {
 			const modelsData = data3mf.model;
 			const modelRels = data3mf.modelRels;
 			const objects = {};
-			const modelsKeys = Object.keys( modelsData );
+			const modelsKeys = Object.keys( modelsData ).reverse();
```

Comment:
I'm afraid I don't understand why this change is a secure fix. When order of model entries is random, how can always reversing the array order fix all possible assets?